src/authenticator.rs

```diff
@@ -98,23 +106,23 @@ pub struct Request {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
-pub struct Response {
+/// Basic response coming from the NR token retrieval endpoint
+pub struct TokenRetrievalResponse {
```

Comment:
Would make sense to rename the Request for consistency too?

Reply:
Done in https://github.com/newrelic/newrelic-auth-rs/pull/31/commits/7d7cdec77994c3b601b50aa9031b3696da9ea714!